frontend/src/types.ts

```diff
@@ -956,6 +956,10 @@ export interface TrendResult {
 
 export interface TrendResultWithAggregate extends TrendResult {
     aggregated_value: number
+    persons: {
+        url: string
+        filter: Partial<FilterType>
```

Comment:
Maybe filter is relevant to not just persons but the aggregate_value it's self? i.e. maybe it could live one level up?

Reply:
For aggregated results there doesn't need to be this nesting but i'm adding this for now so this payload matches the other person payloads